server/bleep/migrations/20230613143506_file-cache.sql

```diff
@@ -1,6 +1,12 @@
 -- Add migration script here
 CREATE TABLE file_cache (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    repo_ref TEXT NOT NULL,
-    cache_hash TEXT NOT NULL
+    cache_hash TEXT PRIMARY KEY NOT NULL,
+    repo_ref TEXT NOT NULL
+);
+
+CREATE TABLE chunk_cache (
+    chunk_hash TEXT NOT NULL,
+    file_hash TEXT NOT NULL,
```

Comment:
is this a foreign key to the `file_cache` table?

Reply:
No, these are different. This is also why we need the `repo_ref`, so we can cross-reference which ones we need to keep when a repo is being deleted.